xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -650,11 +611,9 @@ float CGUIFontTTF::GetLineHeight(float lineSpacing) const
   return 0.0f;
 }
 
-const unsigned int CGUIFontTTF::spacing_between_characters_in_texture = 1;
-
 unsigned int CGUIFontTTF::GetTextureLineHeight() const
 {
-  return m_cellHeight + spacing_between_characters_in_texture;
+  return m_cellHeight + SPACING_BETWEEN_CHARACTERS_IN_TEXTURE;
 }
 
 std::vector<CGUIFontTTF::Glyph> CGUIFontTTF::GetHarfBuzzShapedGlyphs(const vecText& text)
```

Comment:
i cannot comment below this line but there are others loops `unsigned int` should be `size_t`

Reply:
Has changed it with additional commit, only as it uses negative values (-1) and `hb_buffer_add` as `unsigned int` has changed one from `int` to `unsigned int` and about the size it use now a `static_cast`.